addons/mail/static/src/new/views/chatter.xml

```diff
@@ -16,15 +16,34 @@
                     Activities
                 </button>
                 <span class="flex-grow-1"/>
-                <button class="btn px-2" t-att-disabled="!props.resId">
+                <button class="o-mail-chatter-topbar-add-attachments btn px-2" t-att-disabled="!props.resId">
                     <i class="fa fa-paperclip fa-lg"/>
                 </button>
-                <button class="btn px-2" t-att-disabled="!props.resId">
-                    <i class="fa fa-user-o"/>
-                    <span class="ps-1">
-                        <t t-esc="state.followers.length"/>
-                    </span>
-                </button>
+                <div class="o-mail-chatter-topbar-follower-list d-flex position-relative" t-ref="follower-list">
+                    <button class="o-mail-chatter-topbar-follower-list-button btn btn-light px-2" t-att-disabled="isDisabled" t-att-aria-expanded="state.isFollowerDropdownOpen ? 'true' : 'false'" aria-label="Show Followers" t-on-click="onClickFollowersButton">
+                        <i class="fa fa-user-o" role="img"/>
+                        <span class="o-mail-chatter-topbar-followers-count ps-1" t-esc="state.followers.length"/>
+                    </button>
+                    <div t-if="state.isFollowerDropdownOpen" class="o-mail-chatter-topbar-follower-list-dropdown o-dropdown-menu dropdown-menu-end flex-column d-flex" role="menu">
```

Comment:
Could have used `<Dropdown>` component, so visual/behaviour is more consistent with other dropdowns.
Also you wouldn't need to introduce `state.isFollowerDropdownOpen`

Reply:
Ok, can do as follow up

Reply:
https://github.com/odoo-dev/odoo/pull/1490